package.json

```diff
@@ -5,7 +5,7 @@
   "engines": {
     "node": "20"
   },
-  "packageManager": "pnpm@9.6.0+sha256.dae0f7e822c56b20979bb5965e3b73b8bdabb6b8b8ef121da6d857508599ca35",
+  "packageManager": "pnpm@9.13.2+sha512.88c9c3864450350e65a33587ab801acf946d7c814ed1134da4a924f6df5a2120fd36b46aab68f7cd1d413149112d53c7db3a4136624cfd00ff1846a0c6cef48a",
```

Comment:
Is it necessary to update this?

Reply:
@taroj1205 
There is no particular meaning either.